Source/Utils/WarpXUtil.cpp

```diff
@@ -194,8 +231,11 @@ void ConvertLabParamsToBoost()
     {
         if (boost_direction[dim_map[idim]]) {
             amrex::Real convert_factor;
-            // Assume that the window travels with speed +c
-            convert_factor = 1._rt/( gamma_boost * ( 1 - beta_boost ) );
+            amrex::Real beta_window = beta_boost;
```

Comment:
Shouldn't this be 
```
amrex::Real beta_window = 0;
```
i.e. if `do_moving_window` is false, the velocity of the moving window is effectively 0?

Reply:
Apologies, I got confused. After more thought, I agree that this should be `amrex::Real beta_window = beta_boost;` in the absence of an explicit moving window.